packages/sdk/contractkit/src/contract-cache.ts

```diff
@@ -169,6 +172,14 @@ export class WrapperCache implements ContractCacheType {
     return this.getContract(CeloContract.Freezer)
   }
 
+  getFederatedAttestations() {
+    return this.getContract(CeloContract.FederatedAttestations)
+  }
+
+  // getFeeCurrencyWhitelist() {
```

Comment:
looks like this `getFeeCurrencyWhitelist` stuff is a weird merge conflict with master somehow, this was commented out when we started working on ASv2 but was deleted in the meantime on master -- can you delete it here?